src/test/ui/lifetimes/lifetime-doesnt-live-long-enough.stderr

```diff
@@ -80,7 +80,7 @@ error[E0309]: the parameter type `M` may not live long enough
   --> $DIR/lifetime-doesnt-live-long-enough.rs:44:5
    |
 LL |       fn generic_in_child<'a, 'b, L: X<&'a Nested<M>>, M: 'b>() {
-   |       ^                                                -- help: consider adding an explicit lifetime bound `M: 'a`...
+   |       ^                                                --- help: consider adding an explicit lifetime bound `M: 'a`...
```

Comment:
This is perhaps suboptimal, but it should be fixed with https://github.com/rust-lang/rust/issues/65286.